pkgs/http/lib/src/request.dart

```diff
@@ -14,6 +14,22 @@ import 'utils.dart';
 
 /// An HTTP request where the entire request body is known in advance.
 class Request extends BaseRequest {
+  /// Whether the given MIME type should have a 'charset' parameter.
+  bool _shouldHaveCharset(MediaType? contentType) =>
```

Comment:
Should this be a `static` (it does not actually use any `Request` state?) or maybe even global function - rather than being nested in `Request`?

Reply:
I made it `static`.